tests/run-make/alloc-no-oom-handling/rmake.rs

```diff
@@ -0,0 +1,20 @@
+// ignore-tidy-linelength
+
+extern crate run_make_support;
+
+use run_make_support::rustc;
+use std::path::PathBuf;
+
+fn main() {
+    let lib_path = PathBuf::from("../../../library/alloc/src/lib.rs");
```

Comment:
You can avoid path separators (`PathBuf::from` does not attempt to validate its string source, nor does it convert path separators AFAIK) via:

```rs
let lib_path = ["..", "..", "..", "library", "alloc", "src", "lib.rs"].iter().collect();
```

This would internally use the platform separator for the target the test is running on, and when you do `path.to_str().unwrap()` you would get back a path string with the correct platform separator, for any executable (including the ones that don't try to normalize paths).

Reply:
Should I also try to the absolute path here using tmp dir i think or the relative path is okay?

Reply:
Here tmpdir is irrelevant because we're actually referring to the stdlib sources, not compiled libs

Reply:
Makes sense thanks